tests/test_queue.py

```diff
@@ -306,6 +308,20 @@ def test_dequeue_any_ignores_nonexisting_jobs(self):
         )
         self.assertEqual(q.count, 0)
 
+    def test_redis_mode(self):
+        q = Queue()
+        ret = q.redis_mode(self.testconn)
+        logger.info('wqlu redis mode is {}'.format(ret))
+        self.assertIsNotNone(ret)
+        self.assertEqual(ret, 'standalone')
+
+    def test_is_redis_cluster(self):
+        q = Queue()
+        ret = q.is_redis_cluster(self.testconn)
+        logger.info('wqlu is redis cluster {}'.format(ret))
+        self.assertIsNotNone(ret)
```

Comment:
We can omit this line since `self.assertEqual(ret, False)` implies that `ret` is not `None`.

Reply:
👌, Will change later.